packages/expo-ads-admob/README.md

```diff
@@ -34,12 +34,20 @@ In your app's `Info.plist` file, add a `GADApplicationIdentifier` key with a str
 
 ### Configure for Android
 
-In your app's `AndroidManifest.xml` file, add a `meta-data` element with a string value of your AdMob app ID, as shown in [Google's Mobile Ads SDK (Android) documentation](https://developers.google.com/admob/android/quick-start#update_your_androidmanifestxml).
+Ensure that there is a `meta-data` element inside the `application` node inside `AndroidManifest.xml` file (located typically under `/android/app/src/main/AndroidManifest.xml`) with `android:name` of `"com.google.android.gms.ads.APPLICATION_ID"` and a value of your AdMob App ID. Google's Mobile Ads SDK documentation shows precisely how to do this [here](https://developers.google.com/admob/android/quick-start#update_your_androidmanifestxml). In the end your `AndroidManifest.xml` should look more or less like this:
```

Comment:
Small grammar and casing nits: "typically located" and "app ID", per the documentation